storage.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /user/{uid}/{imageName} {
+      allow read, write: if request.auth!=null && request.auth.uid == uid;
+    }
```

Comment:
Shouldn't we add the rules that were already made for cloud.firestore?

Reply:
The users are uploading the images only in /user/{uid}/imageName.jpg so I think that's enough